proposals/NNNN-8-bit-integers.md

```diff
@@ -0,0 +1,65 @@
+<!-- {% raw %} -->
+
+# 8-bit integer support
+
+* Proposal: [0099](NNNN-8-bit-integers.md)
+* Author(s): [Matthäus Chajdas](https://github.com/anteru)
+* Sponsor: [Chris Bieneman](https://github.com/llvm-beanz)
+* Status: **Under Consideration**
+* Planned Version: SM 6.x
+* Issues: [HLSL 337](https://github.com/microsoft/hlsl-specs/issues/337)
+
+## Introduction
+
+This proposal introduces 8-bit integer types to HLSL, to align it more closely with C/C++ and other languages, and allow tight interop with the 8-bit types used in the cooperative vector proposal.
```

Comment:
As the cooperative vector proposal has been rejected it might be better reframe this proposal.  Does it still make sense when combined with the linear algebra matrix proposal?  Does this proposal bring enough benefits on its own without tying it to another one?